### ISSUE TITLE
Checks not triggering for PRs

### ISSUE THREAD

**Issue by seabbs** [epinowcast/epinowcast#306]
It seems like checks aren't triggering for @adrian-lison in PRs. Not at all clear why this might be the case. They do trigger for me. Could potentially be something in the branch rules for `main`, a bug, or something to do with @adrian-lison setup?

**Comment by Bisaloo** [comment]
Do you have a link to an example? A typical situation that can cause are merge conflicts arising from an outdated branch.

**Comment by seabbs** [comment]
This PR: https://github.com/epinowcast/epinowcast/pull/224

They only run for me and not @adrian-lison.

**Comment by Bisaloo** [comment]
So, yes, unless you have a different example, I think it's what I suggested.

The issue is not that they run for you and not Adrian. This is just a coincidence. They started running once conflicts were resolved in [`85698a0` (#224)](https://github.com/epinowcast/epinowcast/pull/224/commits/85698a017e3799dde5a00c8952220d04d9b49a4c). The same situation is visible earlier in the PR, even when just Adrian is involved with checks starting to run again after [`4dfcefb` (#224)](https://github.com/epinowcast/epinowcast/pull/224/commits/4dfcefba4e4d0ab944eb3ce3cef3c30f63c1d871).
